bin/ruby-build

```diff
@@ -918,6 +919,28 @@ use_homebrew_yaml() {
   fi
 }
 
+use_freebsd_pkg() {
+  # check if FreeBSD
+  if [ "FreeBSD" = "$(uname -s)" ]; then
+      # use openssl if installed from Ports Collection
+      if [ -f /usr/local/include/openssl/ssl.h ]; then
+	  package_option ruby configure --with-openssl-dir="/usr/local"
+      fi
+
+      # check if 11-R or later
+      if [ "$(uname -r | sed 's/[^[:digit:]].*//')" -ge 11 ]; then
```

Comment:
I feel that this approach of extracting the release number is brittle. Instead, if the output of `uname -r` is always going to be a number followed by a dash, I suggest the following approach:

```sh
release="$(uname -r)"
release="${release%%-*}"
[ "$release" -ge 11 ]
```